packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -159,6 +160,15 @@ class Plugin implements CLNPlugin {
     throw Exception("Method with name $name not found!");
   }
 
+  Future<void> _handlingNotification(
+      {required String method, required Map<String, Object> request}) async {
+    if (notifications.containsKey(method)) {
+      var notification = notifications[method]!;
+      await notification.call(this, request);
+    }
+    throw Exception("Notification with name $method is not found!");
+  }
```

Comment:
```suggestion
      await notification.call(this, request);
      return;
    }
    throw Exception("Notification with name $method is not found!");
  }
```
otherwise, this always returns an exception!